doc/man3/SSL_shutdown.pod

```diff
@@ -80,11 +80,12 @@ It should only be done when the peer has a way to make sure all
 data has been received and doesn't wait for the close_notify alert
 message, otherwise an unexpected EOF will be reported.
 
-There are implementations that do not send the required close_notify alert.
-If there is a need to communicate with such an implementation, and it's clear
-that all data has been received, do not wait for the peer's close_notify alert.
-Waiting for the close_notify alert when the peer just closes the connection will
-result in an error being generated.
+There are implementations that do not send the required close_notify alert. If
+there is a need to communicate with such an implementation, and it's clear that
+all data has been received, do not wait for the peer's close_notify alert.
```

Comment:
It's unclear to me why you reformat the above.

Note that the old one had the "." at the end of a line, and then the next sentence on the next line. pod2man recognized that "." at the end of a line. At least some people will format all the manpages like that, but I don't think we have a general style guide about it (yet).

Reply:
Removed the reformatting. Thanks!